External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -2832,6 +2832,8 @@ private static void GenerateFunctionJob(GeneratorJobType job, ScintillaControl s
             }
             if (caller?.Parameters != null && caller.Parameters.Count > 0)
             {
+                Func<string, string> cleanType = null;
+                cleanType = s => s.StartsWith("(") && s.EndsWith(')') ? cleanType(s.Trim('(', ')')) : s;

```

Comment:
I think that if there is an incorrect type here, then the error was earlier, when determining the type